===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-alias.c,v
retrieving revision 1.17
retrieving revision 1.18
diff -u -r1.17 -r1.18
--- libc/nis/nss_nisplus/nisplus-alias.c	2005/12/03 17:01:45	1.17
+++ libc/nis/nss_nisplus/nisplus-alias.c	2006/03/25 20:59:19	1.18
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997,1998,2001,2002,2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 1998, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -51,7 +52,7 @@
       static const char prefix[] = "mail_aliases.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-ethers.c,v
retrieving revision 1.21
retrieving revision 1.22
diff -u -r1.21 -r1.22
--- libc/nis/nss_nisplus/nisplus-ethers.c	2005/12/03 21:45:55	1.21
+++ libc/nis/nss_nisplus/nisplus-ethers.c	2006/03/25 20:59:19	1.22
@@ -1,4 +1,4 @@
-/* Copyright (C) 1997,1998,2000-2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997,1998,2000-2003,2005,2006 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@suse.de>, 1997.
 
@@ -94,7 +94,7 @@
       static const char prefix[] = "ethers.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-grp.c,v
retrieving revision 1.18
retrieving revision 1.19
diff -u -r1.18 -r1.19
--- libc/nis/nss_nisplus/nisplus-grp.c	2005/12/03 05:04:16	1.18
+++ libc/nis/nss_nisplus/nisplus-grp.c	2006/03/25 20:59:19	1.19
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997, 2001, 2002, 2003, 2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -47,7 +48,7 @@
       static const char prefix[] = "group.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-hosts.c,v
retrieving revision 1.28
retrieving revision 1.29
diff -u -r1.28 -r1.29
--- libc/nis/nss_nisplus/nisplus-hosts.c	2005/12/03 17:47:33	1.28
+++ libc/nis/nss_nisplus/nisplus-hosts.c	2006/03/25 20:59:19	1.29
@@ -1,4 +1,4 @@
-/* Copyright (C) 1997-2002, 2003, 2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997-2002, 2003, 2005, 2006 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@suse.de>, 1997.
 
@@ -186,7 +186,7 @@
       static const char prefix[] = "hosts.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-network.c,v
retrieving revision 1.21
retrieving revision 1.22
diff -u -r1.21 -r1.22
--- libc/nis/nss_nisplus/nisplus-network.c	2005/12/03 22:35:12	1.21
+++ libc/nis/nss_nisplus/nisplus-network.c	2006/03/25 20:59:19	1.22
@@ -1,4 +1,4 @@
-/* Copyright (C) 1997,1998,2000-2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997,1998,2000-2003,2005,2006 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -148,7 +148,7 @@
       static const char prefix[] = "networks.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-proto.c,v
retrieving revision 1.20
retrieving revision 1.21
diff -u -r1.20 -r1.21
--- libc/nis/nss_nisplus/nisplus-proto.c	2005/12/03 19:33:09	1.20
+++ libc/nis/nss_nisplus/nisplus-proto.c	2006/03/25 20:59:19	1.21
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997,1998,2001,2002,2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 1998, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -147,7 +148,7 @@
       static const char prefix[] = "protocols.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-pwd.c,v
retrieving revision 1.20
retrieving revision 1.21
diff -u -r1.20 -r1.21
--- libc/nis/nss_nisplus/nisplus-pwd.c	2005/12/03 20:39:34	1.20
+++ libc/nis/nss_nisplus/nisplus-pwd.c	2006/03/25 20:59:19	1.21
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997,1999,2001,2002,2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 1999, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -44,7 +45,7 @@
       static const char prefix[] = "passwd.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (pwd_tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-rpc.c,v
retrieving revision 1.20
retrieving revision 1.21
diff -u -r1.20 -r1.21
--- libc/nis/nss_nisplus/nisplus-rpc.c	2005/12/03 19:54:38	1.20
+++ libc/nis/nss_nisplus/nisplus-rpc.c	2006/03/25 20:59:19	1.21
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997,1998,2001,2002,2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 1998, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@vt.uni-paderborn.de>, 1997.
 
@@ -144,7 +145,7 @@
       static const char prefix[] = "rpc.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
===================================================================
RCS file: /cvs/glibc/libc/nis/nss_nisplus/nisplus-service.c,v
retrieving revision 1.22
retrieving revision 1.23
diff -u -r1.22 -r1.23
--- libc/nis/nss_nisplus/nisplus-service.c	2005/12/03 20:11:34	1.22
+++ libc/nis/nss_nisplus/nisplus-service.c	2006/03/25 20:59:19	1.23
@@ -1,4 +1,5 @@
-/* Copyright (C) 1997-1999,2001,2002,2003,2005 Free Software Foundation, Inc.
+/* Copyright (C) 1997, 1998, 1999, 2001, 2002, 2003, 2005, 2006
+   Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Thorsten Kukuk <kukuk@suse.de>, 1997.
 
@@ -151,7 +152,7 @@
       static const char prefix[] = "services.org_dir.";
 
       char *p = malloc (sizeof (prefix) + local_dir_len);
-      if (tablename_val == NULL)
+      if (p == NULL)
 	{
 	  *errnop = errno;
 	  return NSS_STATUS_TRYAGAIN;
